Precheck view and note the programmatic req

nickgros opened the pull request:
Appearance in Synapse.org: 

![image](https://user-images.githubusercontent.com/17580037/108566477-7e263b00-72d4-11eb-9863-7837f8dbdfa2.png)

jay-hodgson commented:
I think the descriptions of each permission in parenthesis is suboptimal.  Would rather see them in subdued text.  This also solves the parens inside parens, where it starts to look like lisp :)
This comment does not block the merge.  Here's an example of what I'm thinking of though...
https://forum.gitlab.com/t/question-scopes-which-scopes-required-for-project-exporting/29542

nickgros commented:
I like your suggestion! Here's the new version:

![image](https://user-images.githubusercontent.com/17580037/108734391-6e3d6f80-74fd-11eb-9dd7-a2c3b239349a.png)

I also cleaned up some of the CSS and refactored a tiny amount of component behavior by introducing useListView (which wasn't added when this component was created)

jay-hodgson reviewed (approved):
Looks great!
